packages/fake-driver/lib/driver.js

```diff
@@ -1,6 +1,6 @@
 import B from 'bluebird';
 import {BaseDriver, errors} from 'appium/driver';
-import {deprecatedCommandsLogged} from '@appium/base-driver/build/lib/protocol/protocol';
+import {deprecatedCommandsLogged} from '@appium/base-driver/lib/protocol/protocol';
```

Comment:
If we need `deprecatedCommandsLogged` outside of `@appium/base-driver`, then it should be exported from `@appium/base-driver`.  that will make it so it can be imported from `appium/driver`, as it should be.  this means we are making `deprecatedCommandsLogged` a public API.

please export this function from `@appium/base-driver/lib/index.js`

Reply:
(note: importing from `@appium/base-driver` or `@appium/support` is discouraged in drivers, since we do not explicitly depend on these packages)

Reply:
I have reverted the change. this anyway is not related to the current PR, so should not be mixed. 
Up to @jlipps to decide what to so with this import